frontend/src/common/entities.ts

```diff
@@ -102,7 +102,7 @@ export interface Collection {
   updated_at: number;
   publisher_metadata: PublisherMetadata;
   revision_diff: boolean;
-  summaryCitation?: string;
+  summaryCitation: string;
```

Comment:
Updated `summaryCitation` to be required with a "no value" value of `NO_PUBLICATION` rather than `""`. This simplifies the both the filter and filter display functionality; keeping "no value" as `""` would require the core filter functionality to be updated to handle falsey values.